src/game.cpp

```diff
@@ -2851,6 +2850,16 @@ void Game::toggleCinematic(float *statustext_time)
 	statustext = msg[cinematic];
 }
 
+// Add WoW-style autorun by toggling continuous forward.
+void Game::toggleAutorun(float *statustext_time)
+{
+	static const wchar_t *msg[] = { L"autorun disabled", L"autorun enabled" };
+	bool autorun = !g_settings->getBool("continuous_forward");
+	g_settings->set("continuous_forward", bool_to_cstr(autorun));
+
+	*statustext_time = 0;
+	statustext = msg[autorun];
```

Comment:
or name it like `is_autorun` or `autorun_enabled` so it's more obvious that it's a flag.